### DISCUSSION TITLE
Vevői app

### DISCUSSION THREAD

**Discussion by MicraMetripond** [nav-gov-hu/eRECEIPT#22] [Kérdések és válaszok]
"Adat helyreállítás funkció az elérési adatok alapján." - Ez hogyan történik ha az applikációban a felhasználó nincs regisztrálva és az applikáció nem tárolhat semmilyen adatot?

**Comment by renced42** [comment]
Kedves @MicraMetripond 

A vevői applikációnak a titkosító kulcspárokat és egyéb meta információkat kell mentenie a helyreállításhoz. Ezen adatok összessége alapján a nyugtatárból visszaállíthatók a nyugták.
